SlicerCART/documentation/quickstart.md

```diff
@@ -0,0 +1,128 @@
+appreciated for 1) Installation steps 2) Loading datasets 3) General Insight.
+
+
+### Before starting
+
+You should have:
+* Successfully installed 3D Slicer
+* Sucessfully installed SlicerCART
+
+Please review [Installation Steps](installation.md#installation-steps) if those criteria are not 
+fullfilled.
+
+
+
+### Getting started
+
+**Example use case:** viewing different volumes in a given dataset and 
+perform manual segmentations
+
+
+
+###### 1. Open 3D Slicer
+###### 2. Launch SlicerCART
+###### 3. Select "New Configuration", and click on Next
+When the module is loaded, a pop-up window appears and allow the user to 
+customize SlicerCART settings. 
+
+![Alt Text](images/select_configuration_popup.png)
+
+###### 4. Specify the appropriate dataset options and tasks, and then click Apply
+For example, select if you will complete segmentation and/or classification 
+tasks, the modality (takes only 1 for now), if you impose BIDS (will pass a 
+BIDS validator test before loading volumes), dataset volumes extensions (i.e.
+Input File Extension), etc.
+
+By default, some segmentation labels will be used. We recommend for now to 
+use those (soon, it should be able to modify them by clicking on Configure 
+Segmentation). The same applies for "Configure Classification".
+
+![Alt Text](images/select_configuration_module.png)
+
+
+###### 5. Select Volumes folder, and specify annotator information 
+* Select the folder that contains the images that you want to process (if 
+* BIDS folder, will not consider images in  derivatives), manipulate, view, etc.
+
+* Specify the Annotator name, degree and revision step (all are mandatory 
+*    for saving functions).
+
+![Alt Text](images/folder_and_name_to_use.png)
+![Alt Text](images/example_loading_cases_ui.png)
+
+
+
+N.B. If loading cases in the UI fails, please open an issue on the Github 
+   repository or ask a team member. If this steps has not succeed, you will 
+   not be able to use SlicerCART (e.g. imaging format incompatibility).
+
+
+###### 6. Select Output folder
+Select the folder where output data (e.g. segmentation masks, statistics) 
+will be saved. For now, it must be empty.
+![Alt Text](images/select_output_folder.png)
+
+###### 7. Display Volume to Segment
+Click on the case in Case list you want display and/or perform segmentation, 
+so it will be shown in the Slicer Viewer.
+![Alt Text](images/select_volume_to_segment.png)
+
+###### 8. Start Segmentation
+Click on the case you want to 
+Start segmentation by clicking on:
+- SegmentEditor: will open the default segment editor of 3D Slicer
+- Paint: will make the user able to paint the **first** mask label
+- Erase: will make the user able to erase current segment label
+
+![Alt Text](images/perform_segmentation.png)
+
+###### 9. Save Segmentation
+Once segmentation is completed, click on Save segmentation to save the 
+segmentation mask in the output folder. Note that a .csv file will be 
```

Comment:
saving a JSON that is BIDS compatible with derivatives would be much better-- pls open an issue on this